index.js

```diff
@@ -109,6 +109,9 @@ Gleak.prototype.whitelist = [
   , global
   , GLOBAL
   , root
+// io.js
+  , clearImmediate
+  , setImmediate
```

Comment:
we'll need to fix the branching logic to check the semver major version number and include a new branch for io release numbers